### ISSUE TITLE
ScreenReader add on not working

### ISSUE THREAD

**Issue by rfx91** [FrDH/mmenu-js#1235]
I have Mmenu up and running on my local app and I'm noticing that there is no screen reader support. Running NVDA and using the keyboard doesn't allow the user to navigate within the menu once opened, there are no aria labels, and there is no announcement that a menu was opened. The docs say that screen reader support with aria is enabled by default. Do you know what's up with that?

**Comment by FrDH** [comment]
Keyboard navigation should be enabled using the [keyboard navigation add-on](https://mmenujs.com/docs/addons/keyboard-navigation.html) (it'll be enabled by default in the next major release though).

Better ARIA support is also added in the next major release.

**Comment by nanna-dk** [comment]
Great news. When is next major release?
